source/index.js

```diff
@@ -24,10 +24,9 @@ const defaults = {
 				504
 			]
 		},
-		cache: false,
 		decompress: true,
-		useElectronNet: false,
```

Comment:
Not clear why you removed the defaults for `useElectronNet` and `cache`?

Reply:
It's not needed. There are many options which are `false` by default and those aren't here :) IMO we should only set defaults which point to `true`, it's more clear. Others are `false` implicitly.

Reply:
I think it's better to define all defaults upfront? What if the check is `foo == false`, then it will never match, since the default is not there, and the user would only pass in `true`.